lib/shared/bucketing-assembly-script/assembly/types/target.ts

```diff
@@ -18,6 +18,7 @@ export class Target extends JSON.Value {
     readonly _audience: Audience
     readonly rollout: Rollout | null
     readonly distribution: TargetDistribution[]
+    readonly bucketingKey: string
```

Comment:
does this class need to be updated, or does it make more sense to use the V2 classes I created? I did it that way so that we could validate v1 configs against the v1 classes in the config service. 

Technically this wont fail for v1 configs since they all have bucketing key anyways, so its more of a philosophical question than anything